docs/plugins/minify-simplify.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: Minify simplify
+description: This plugin will transform code in mainly two ways - 1) Reduce as much statements as possible into expressions, and 2) Make expressions as uniform as possible for better compressibility
+permalink: /docs/plugins/minify-simplify/
+package: babel-plugin-minify-simplify
+---
+
+{% include package_readme.html from="babili" %}
```

Comment:
![minify-simplify](https://cloud.githubusercontent.com/assets/1493671/20865451/7d71fe80-ba12-11e6-8177-ec2953f78914.png)

I'm not sure why this happens, the README seems good: https://github.com/babel/babili/blob/master/packages/babel-plugin-minify-simplify/README.md